commodore/postprocess/builtin_filters.py

```diff
@@ -31,7 +31,9 @@ def _builtin_filter_helm_namespace(
         create_namespace = "true" if create_namespace else "false"
     exclude_objects = kwargs.get("exclude_objects", [])
     exclude_objects = "|".join([json.dumps(e) for e in exclude_objects])
-    output_dir = _output_dir(work_dir, component.name, path)
+    # XXX: we pass "" for the component name here, as we already patch
+    # `path` to contain the output dir in postprocess/__init__.py
```

Comment:
What's with the XXX?

Reply:
Changed to `NOTE:` in the latest version